internal/scorecard/scorecard_output.go

```diff
@@ -25,26 +25,26 @@ import (
 	scapiv1alpha2 "github.com/operator-framework/operator-sdk/pkg/apis/scorecard/v1alpha2"
 )
 
-func printPluginOutputs(version string, pluginOutputs []scapiv1alpha1.ScorecardOutput) error {
+func printPluginOutputs(cfg Config, pluginOutputs []scapiv1alpha1.ScorecardOutput) error {
```

Comment:
This could be a method on `Conifg`.

Reply:
good suggestion, the change will be in the next commit.